app/views/documents/care_provider_statements/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for :form_question, "Attach your statements from childcare facilities or individuals who provided care." %>
+
+<% content_for :form_help_text do %>
+  <strong>You might have this statement</strong>
+  because you said that you paid for child or dependent care.
```

Comment:
can this also be senior care? if so should we list it out explicitly? "dependent care" may still be too vague